Extend documentation and move to precice.org (#6)
precice/fmi-runner: README.md

# preCICE-FMI Runner

A tool to couple FMU models with other simulation tools via the coupling library [preCICE](https://precice.org/).

> ## Start here

Reviewer: ```suggestion
```

> The main documentation is on the preCICE website **add final link**.

Reviewer: ```suggestion
The main documentation is rendered on the [preCICE website](https://precice.org/tooling-fmi-runner.html).
```

> Please report any bugs or issues [here](https://github.com/precice/fmi-runner/issues) and give us feedback through [one of our community channels](https://precice.org/community-channels.html).

Reviewer: ```suggestion
Please report [issues](https://github.com/precice/fmi-runner/issues) and give us feedback through [one of our community channels](https://precice.org/community-channels.html).
```

## References

The concept and design of the FMI runner is described in:

> L. Willeke, [A preCICE-FMI Runner to couple controller models to PDEs]( http://dx.doi.org/10.18419/opus-13130), Master Thesis, University of Stuttgart, 2023

Reviewer: ```suggestion
L. Willeke, [A preCICE-FMI Runner to couple controller models to PDEs](http://dx.doi.org/10.18419/opus-13130), Master Thesis, University of Stuttgart, 2023
```
